extensions/exposition/features/steps/Workspace.ts

```diff
@@ -23,7 +24,7 @@ export class Workspace {
     const source = join(__dirname, 'components', name)
```

Comment:
This is `posix.join`, it's not compatible with Windows.

Because of subdirectory `import`, it is implicit, easy to make such a mistake, hard to read and find it on the review.

Always use explicit calls like `posix.join()`.